tests/foreman/virtwho/api/test_nutanix.py

```diff
@@ -193,7 +190,96 @@ def test_positive_hypervisor_id_option(
                 command, form_data['hypervisor_type'], org=default_org.label
             )
             assert get_configure_option('hypervisor_id', config_file) == value
-        virtwho_config.delete()
-        assert not target_sat.api.VirtWhoConfig().search(
-            query={'search': f"name={form_data['name']}"}
+
+    @pytest.mark.tier2
+    @pytest.mark.parametrize('deploy_type', ['id', 'script'])
+    def test_positive_prism_central_deploy_configure_by_id_script(
+        self, default_org, form_data, target_sat, deploy_type
+    ):
+        """Verify "POST /foreman_virt_who_configure/api/v2/configs" on nutanix prism central mode
+
+        :id: e7652f64-eaf8-45a5-ac01-eb40d53b6603
+
+        :expectedresults:
+            Config can be created and deployed
+            The prism_central has been set in /etc/virt-who.d/vir-who.conf file
+
+        :CaseLevel: Integration
+
+        :CaseImportance: High
+        """
+        form_data['prism_flavor'] = "central"
+        virtwho_config = target_sat.api.VirtWhoConfig(**form_data).create()
+        assert virtwho_config.status == 'unknown'
+        if deploy_type == "id":
+            command = get_configure_command(virtwho_config.id, default_org.name)
+            hypervisor_name, guest_name = deploy_configure_by_command(
+                command, form_data['hypervisor_type'], debug=True, org=default_org.label
+            )
+        elif deploy_type == "script":
+            script = virtwho_config.deploy_script()
+            hypervisor_name, guest_name = deploy_configure_by_script(
+                script['virt_who_config_script'],
+                form_data['hypervisor_type'],
+                debug=True,
+                org=default_org.label,
+            )
+        # Check the option "prism_central=true" should be set in etc/virt-who.d/virt-who.conf
+        config_file = get_configure_file(virtwho_config.id)
+        assert get_configure_option("prism_central", config_file) == 'true'
+        virt_who_instance = (
+            target_sat.api.VirtWhoConfig()
+            .search(query={'search': f'name={virtwho_config.name}'})[0]
+            .status
         )
+        assert virt_who_instance == 'ok'
+        hosts = [
+            (
+                hypervisor_name,
+                f'product_id={settings.virtwho.sku.vdc_physical} and type=NORMAL',
+            ),
+            (
+                guest_name,
+                f'product_id={settings.virtwho.sku.vdc_physical} and type=STACK_DERIVED',
+            ),
+        ]
+        for hostname, sku in hosts:
+            host = target_sat.cli.Host.list({'search': hostname})[0]
+            subscriptions = target_sat.cli.Subscription.list(
+                {'organization': default_org.name, 'search': sku}
+            )
```

Comment:
I will suggest using APIs here to get hosts and subscriptions

Reply:
> I will suggest using APIs here to get hosts and subscriptions

@Gauravtalreja1 thanks for your suggestion. host list can using apis "target_sat.api.Host().search(query={'search': hostname})[0].read_json()", Do you know the subscription list or subscription or search usage "target_sat.api.HostSubscription".

Reply:
@yanpliu I see, we do not have a function for Subscription.list in nailgun yet, so you can contribute that functionality if you'd like, otherwise I wouldn't block this PR.
https://github.com/SatelliteQE/nailgun/blob/master/nailgun/entities.py#L7190

Reply:
> @yanpliu I see, we do not have a function for Subscription.list in nailgun yet, so you can contribute that functionality if you'd like, otherwise I wouldn't block this PR. https://github.com/SatelliteQE/nailgun/blob/master/nailgun/entities.py#L7190

@Gauravtalreja1 Have already using API to get hosts and subscriptions https://github.com/SatelliteQE/robottelo/pull/10017/commits/f07add9c0137d1b4e3c0d667401c1078edd2da90

Also submit PR for nailgun to support this function.
https://github.com/SatelliteQE/nailgun/pull/843

Reply:
@yanpliu Thanks for the nailgun PR, could you update a latest test results?

Reply:
> @yanpliu Thanks for the nailgun PR, could you update a latest test results?

@Gauravtalreja1 Of course.
This is the latest test results:
```
(robottelo_vv) [yanpliu@yanpliu robottelo]$ pytest ./tests/foreman/virtwho/api/test_nutanix.py --disable-pytest-warnings -q
......                                                                                                                                                                                                      [100%]
6 passed, 6 deselected, 76 warnings in 538.55s (0:08:58)

```